<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MolWidget</class>
 <widget class="QWidget" name="MolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="molList"/>
   </item>
   <item>
    <widget class="QPushButton" name="atomTableButton">
     <property name="text">
      <string>Atoms</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="atomTable">
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>z</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cellWidgetButton">
     <property name="text">
      <string>Cell Geometry</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="cellContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="cellDimLayout">
        <item>
         <widget class="QLabel" name="cellVecLabel">
          <property name="text">
           <string>Cell vectors:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="cellDimSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="cellDimLabel">
          <property name="text">
           <string>Cell dimension:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cellDimBox">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="cellVecTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>x</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cellEditLayout">
        <item>
         <widget class="QCheckBox" name="cellAllBox">
          <property name="text">
           <string>Apply to all Steps</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="cellEditSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="cellScaleBox">
          <property name="text">
           <string>Scale coordinates with cell</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="kpointStackButton">
     <property name="text">
      <string>K-Points</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="kpointStack">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>atomTableButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>atomTable</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellWidgetButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>cellContainer</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kpointStackButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>kpointStack</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
